services/api/src/utils/errors.js

```diff
@@ -2,6 +2,7 @@ class ApplicationError extends Error {
   constructor(message, status) {
     super(message);
     this.status = status;
+    this.expose = true;
```

Comment:
maybe this is the time we should be reconsidering/removing what I've done with `NotFoundError` etc etc and just say everything will go through `ctx.throw` now?

Reply:
so assuming we want that maybe this entire file can go away?